analyses/survival-analysis/util/survival_models.R

```diff
@@ -0,0 +1,117 @@
+# Functions for conducting survival analyses
+#
+# C. Savonen for ALSF - CCDL
+#
+# 2019
+#
+# Install the packages we need
+if (!("survival" %in% installed.packages())) {
+  install.packages("survival")
+}
+
+if (!("survminer" %in% installed.packages())) {
+  install.packages("survminer")
+}
+
+# Attach this package
+library(survminer)
+
+# Magrittr pipe
+`%>%` <- dplyr::`%>%`
```

Comment:
General question - is there differing behavior in `%>%` between packages? Curious as to the specific reasons for specifying this in all scripts, I'm unaware.

Reply:
> General question - is there differing behavior in `%>%` between packages?

Not that I am aware of.

> Curious as to the specific reasons for specifying this in all scripts, I'm unaware.

We do this to avoid `library(dplyr)` because of the namespace collisions (although it doesn't _usually_ come into play).

Reply:
Makes sense, better safe than sorry. Thanks